src/EnergyPlus/DataSizing.hh

```diff
@@ -455,6 +455,7 @@ namespace DataSizing {
 		bool EMSOverrideDesHeatMassOn; // true if EMS is acting on this structure
 		Real64 EMSValueDesHeatMassFlow; // Value EMS directing to use for Design Heating air mass flow [kg/s]
 		Real64 DesCoolMassFlow; // zone design cooling air mass flow rate [kg/s]
+		Real64 DesCoolMassFlowNoOA; // zone design cooling air mass flow rate with applying MinOA as a limit [kg/s]
```

Comment:
Accidentally messed up this commit - it is a mix of new changes plus merging in develop.  Initial display of this commit shows the new changes mostly, plus conflict resolution changes.